.github/workflows/macos.yml

```diff
@@ -132,11 +132,8 @@ jobs:
             -Ddefault_library=static \
             -Db_asneeded=true \
             -Db_lto=true \
-            -Dtry_sdl2_static=true \
-            -Duse_sdl2_net=false \
+            -Dtry_static_libs=opusfile,png,sdl2,sdl2_net \
```

Comment:
Are `-Ddefault_library=static` and `-Dtry_static_libs= ...` related in some way?

Thinking about folks who prefer static builds, would be easy to provide a value or switch that tries to statically link all supported libraries?

`-Dtry_static_libs=all` == ` -Dtry_static_libs=opusfile,png,sdl2,sdl2_net`

Reply:
Thinking about folks who prefer static builds, would it be possible to provide a value or switch that tries to statically link all supported libraries?

`-Dtry_static_libs=all` == ` -Dtry_static_libs=opusfile,png,sdl2,sdl2_net`

Reply:
@kcgen No; `-Ddefault_library=static` is Meson built-in option. `-Dtry_static_libs` is my invention here.

About adding option `all`… Well, it's possible but it would look ugly in our meson.build file…

Aside for that - it's really, really, really bad idea to try linking SDL2 statically on Linux - it pulls in whole X11 lib and lots and lots of stuff.

I prefer if we kept it as "not fully supported hack". It solves our problem on macOS and that's what I consider important.